src/ScottPlot5/ScottPlot/CoordinateRect.cs

```diff
@@ -39,6 +41,8 @@ public CoordinateRect(double xMin, double xMax, double yMin, double yMax)
 
     public CoordinateRange YRange => new(YMin, YMax);
 
+    public CoordinateRect WithTranslation(Coordinates p) => new(XMin + p.X, XMax + p.X, YMin + p.Y, YMax + p.Y);
```

Comment:
We don't have to implement this now, but an alternative way to do this would be an operator overload with `CoordinateVector`

Actually, should the augment here be a `CoordinateVector` anyway?